health-services/project-factory/src/server/utils/campaignMappingProcessUtils.ts

```
@@ -0,0 +1,689 @@
+        const produceMessage: any = {
+            processName: processNamesConstantsInOrder.mapping,
+            campaignDetailsAndRequestInfo: campaignDetailsAndRequestInfo
+        }
+        await produceModifiedMessages(produceMessage, config.kafka.KAFKA_PROCESS_HANDLER_TOPIC);
+
+        // TODO : Remove confirmation from here
+        // Function to delay execution for a specified time
+        let mappingCreationStatusConfirmed = false;
+        let status = "";
+        const delay = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
+        const processConfirmationAttempts = parseInt(config?.values?.processConfirmationAttempts || "75") || 75;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)